services/common/src/tests/mocks/dataMocks.tsx

```diff
@@ -11,6 +11,7 @@ import {
   IMineDocument,
   IPermitAmendment,
 } from "@mds/common/interfaces";
+import moment from "moment";
```

Comment:
probably don't need the moment import

Reply:
![image](https://github.com/user-attachments/assets/e7d79869-c94f-4518-84f6-adcabcdfb88f)